compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -980,6 +980,24 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
         quant_const(const_node, output_type);
       break;
 
+    case luci::CircleOpcode::PADV2:
+      // Only the third input is quantized
+      // Second input should not be quantized (e.g., paddings)
+      // Quant params propagated from output range to the input
+      input_node = node->arg(2);
```

Comment:
I think the quantization logic needs to be updated as below.

1. We apply `overwrite_quantparam` only to (1) cloned const node or (2) feature map with one successor, because the target node can be used by another node.
(PTAL at `propagate_concat_qparam`)

2. `input` can be a constant tensor. If so, its value should be quantized as well.

Reply:
I've updated logic to follow these rules, now it supports constant input and checks number of successors in case of non-constant node.